easybuild/easyblocks/o/openfoam.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, *args, **kwargs):
 
         self.is_extend = 'extend' in self.name.lower()
         self.is_dot_com = self.looseversion >= LooseVersion('1606')
-        self.is_dot_org = self.looseversion <= LooseVersion('100')
+        self.is_dot_org = self.looseversion < LooseVersion('1606')
```

Comment:
There seems to be no need for this:

```suggestion
        self.is_dot_org = self.looseversion <= LooseVersion('100')